Standards/scs-0001-sovereign-cloud-standards.md

```diff
@@ -0,0 +1,304 @@
+---
+type: Procedural
+status: Draft
+track: Global
+---
+
+# Introduction
+
+The [Sovereign Cloud Stack (SCS)](https://scs.community) provides standards
+for a range of cloud infrastructure types.
+It strives for interoperable and sovereign cloud stacks
+which can be deployed and used by a wide range of organizations and individuals.
+Whereever feasible,
+transparency and openness both in respect to the inner workings of the platforms standardised by SCS,
+as well as the SCS organisation itself
+are a paradigm we intend to live.
+
+# Requirements
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in [RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119).
+
+In addition, "FORBIDDEN" is to be interpreted equivalent to "MUST NOT".
+
+# Motivation and Objectives
+
+TBD.
+
+# Sovereign Cloud Standard documents
+
+One of the main products of the SCS organisation are Sovereign Cloud Standard documents.
+
+## Types of documents
+
+### Procedural
+
+A procedural SCS document describes a process, a policy or a guideline
+to which the SCS community adheres.
+
+### Standard
+
+A standard SCS document describes a technical standard for SCS compliant clouds.
+Note that it may not be necessary for all clouds to implement all standards.
+
+### Decision Record
+
+Sometimes during the development of the SCS standard,
+a complex technical decision needs to be taken,
+which does not directly result in a new standard.
+
+To keep a record of such decisions for future reference,
+they were formerly documented in a format called
+"Architecture Decision Records" (ADRs).
+
+The SCS document format supersedes and integrates such ADRs
+as documents of type `Decision Record`.
```

Comment:
I suggest to rephrase this, as we never really used ADRs, so imho there is no need to refer to them as.
Suggestion:

Sometimes during the development of the SCS standard, a complex technical decision needs to be taken, which does not directly result in a new standard.
The SCS document format integrates the documentation of such decisions with the type `Decision Record`.

Reply:
Addressed.